Mathlib/Data/Complex/Basic.lean

```diff
@@ -670,11 +668,11 @@ theorem normSq_nonneg (z : ℂ) : 0 ≤ normSq z :=
   add_nonneg (mul_self_nonneg _) (mul_self_nonneg _)
 #align complex.norm_sq_nonneg Complex.normSq_nonneg
 
-@[simp]
-theorem range_normSq : range normSq = Ici 0 :=
-  Subset.antisymm (range_subset_iff.2 normSq_nonneg) fun x hx =>
-    ⟨Real.sqrt x, by rw [normSq_ofReal, Real.mul_self_sqrt hx]⟩
-#align complex.range_norm_sq Complex.range_normSq
+-- @[simp]
+-- theorem range_normSq : range normSq = Ici 0 :=
+--   Subset.antisymm (range_subset_iff.2 normSq_nonneg) fun x hx =>
+--     ⟨Real.sqrt x, by rw [normSq_ofReal, Real.mul_self_sqrt hx]⟩
+-- #align complex.range_norm_sq Complex.range_normSq
```

Comment:
It looks like this one has disappeared from the split?

Reply:
Fixed.